src/subcommand/wallet/inscriptions.rs

```diff
@@ -5,6 +5,7 @@ pub struct Output {
   pub inscription: InscriptionId,
   pub location: SatPoint,
   pub explorer: String,
+  pub amount: u64,
```

Comment:
I think `postage` is clearer than `amount` here:

```suggestion
  pub postage: u64,
```